lib/Munin/Master/HTML.pm

```diff
@@ -819,6 +819,11 @@ sub _get_params_services {
 sub _get_params_fields {
 	my ($dbh, $service_id) = @_;
 
+	my $sth_sa = $dbh->prepare_cached("SELECT value FROM service_attr WHERE id = ? and name = ?");
+
+	$sth_sa->execute($service_id, 'graph_period');
```

Comment:
@steveschnepp : any clue to why this prepare/execute returns a empty value?

Compare https://github.com/munin-monitoring/munin/blob/master/lib/Munin/Master/Graph.pm#L227 and the ds query further down.